features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/TimelineView.kt

```diff
@@ -144,6 +144,7 @@ fun TimelineView(
                     key = { timelineItem -> timelineItem.identifier() },
                 ) { timelineItem ->
                     TimelineItemRow(
+                        modifier = Modifier.animateItem(),
```

Comment:
I think this is the cause of a weird animation when expanding/collapsing items:

![ItemAnimation](https://github.com/user-attachments/assets/85bf626c-c872-4cb3-8489-35cd7cb135ed)

Reply:
Oh, I see, maybe I'll have to either change the animation, try to move it down the composable hierarchy or partially/completely remove it.